cgi/login.pl

```diff
@@ -59,15 +59,20 @@
 if ($ENV{'REQUEST_METHOD'} eq 'POST') {
 	my $user_file = "$data_root/users/" . get_string_id_for_lang('no_language', $User_id) . '.sto';
 	my $user_ref = retrieve($user_file);
-	if (not (defined $user_ref)) {
+	if (not(defined $user_ref)) {
```

Comment:
It's a bit strange to remove the space before the "not" operator. Maybe just my habit.